src/balance.rs

```diff
@@ -73,7 +73,8 @@ pub struct BalanceDetails {
 pub enum LightningBalance {
 	/// The channel is not yet closed (or the commitment or closing transaction has not yet
 	/// appeared in a block). The given balance is claimable (less on-chain fees) if the channel is
-	/// force-closed now.
+	/// force-closed now. Values do not take into account any pending splices and are only based
+	/// on the confirmed state of the channel.
```

Comment:
I don't believe this is the case. It *will* take into account a pending splice if it is confirmed but not locked (i.e., doesn't have sufficient number of confirmations).